queries/userinfo.py

```diff
@@ -260,7 +264,9 @@ def _whatsmyaddr_handler(q: Query, ql: str) -> bool:
     r"heimilisfang mitt er (.+)$",
 )
 
-_ADDR_LOOKUP_FAIL = "Ég fann ekki þetta heimilisfang."
+_ADDR_LOOKUP_FAIL = "Ég fann ekki þetta heimilisfang í staðfangaskrá."
+_ADDR_CLIENT_ID_MISSING = "Ég get ekki vistað heimilisfangið því \
```

Comment:
Better to have two strings enclosed by parentheses, rathern than a backslash line continuation